### PR TITLE
djv: Adding OpenEXR Support

### PR DESCRIPTION
**Pull request by ScatteredRay**
## Description of changes

DJV is particularly useful for vfx workflows, where EXRs are common, adding OpenEXR support to the nix buid of djv.

Just adding openexr.dev as an input, and patching FindOpenEXR.cmake to find the correct directory.

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin

- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ScatteredRay** [comment]
Ran: nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD". seemed to succeed..

It says 1 package built, so I'm assuming no packages depend on djv, That would make sense from what I understand of the package.

**Review by JohnRTitor — DISMISSED** [review]
Update commit title, just like you updated PR title.
Then force push.

**Comment by ScatteredRay** [comment]
> Update commit title, just like you updated PR title. Then force push.

Done

File: pkgs/applications/graphics/djv/default.nix [hunk 1]
```diff
@@ -150,6 +151,7 @@ stdenv.mkDerivation rec {
     freetype
     opencolorio_1
     djv-deps
+    openexr.dev
```
Comment (blitz) [inline pkgs/applications/graphics/djv/default.nix:154]:
```suggestion
    openexr
```

(This is completely equivalent.)

Comment (blitz) [reply pkgs/applications/graphics/djv/default.nix:154]: (This also applies to zlib above.)

**Review by blitz — APPROVED** [review]

File: pkgs/applications/graphics/djv/default.nix (hunk 1 above)
Comment (ScatteredRay) [reply pkgs/applications/graphics/djv/default.nix]: :+1:

**Comment by ScatteredRay** [comment]
I'm not sure how to get past the "Merging is blocked" message. It says @JohnRTitor still has a change requested. I have updated the title.

**Comment by JohnRTitor** [comment]
Result of `nixpkgs-review pr 287566` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>djv</li>
  </ul>
</details>

**Review by JohnRTitor — APPROVED** [review]
LGTM

**Pull request merged**